src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -195,13 +201,36 @@ function ObjectSummary(props: ObjectSummaryProps) {
 
     const renderTree = () => {
         return (
-            <div>
+            <div className={b('tree-wrapper')}>
                 <div className={b('tree-header')}>
                     <div className={b('tree-title')}>Navigation</div>
                 </div>
                 <div className={b('tree')}>
                     {tenantData && (
-                        <SchemaNode fullPath={tenantName as string} data={tenantData} isRoot />
+                        <NavigationTree
+                            rootState={{
+                                path: tenantName as string,
+                                name: tenantName as string,
```

Comment:
A question: is tenant name always also tenant path? Or is it better to read the path from the `Path` field?